trace-dispatcher/doc/trace-dispatcher.md

```diff
@@ -108,689 +89,654 @@ data TraceAddBlockEvent blk =
-node's basic information).
+- Each namespace is a terminal and is not a part of another namespace.
 
-DataPoints are implemented as special tracers, which packs the objects into DataPoint
-constructors and require a ToJSON instance for that objects. The set of DataPoints
-provided by the node is structured using the same namespace as metrics and log messages.
-But otherwise DataPoints work independent of tracing, but are written in a local store,
-so the latest value of a particular DataPoint can be queried on demand.
+- Namespaces in the `severityFor`, `privacyFor`, `detailsFor`, `documentFor`, and `metricsDocFor` functions are consistent with the `allNamespaces` definition.
 
-Also, [there is a document](https://github.com/input-output-hk/cardano-node-wiki/wiki/cardano-node-and-DataPoints:-demo)
-describing how to accept DataPoints from an external process.
+- Any namespace in the configuration must be found by a hierarchical lookup in `all namespaces`.
 
-[`demo-acceptor`](https://github.com/intersectmbo/cardano-node/blob/master/cardano-tracer/demo/acceptor.hs)
-application allows to ask for particular DataPoint by its name and display its value.
+## Trace Backends Overview
 
-```haskell
--- A simple dataPointTracer which supports building a namespace
--- `trDataPoint`: shall be the datapoint backend
--- `namesFor`: shall be a function to produce a namespace for every
--- datapoint constructor
-mkDataPointTracer :: forall dp. ToJSON dp
-  => Trace IO DataPoint
-  -> (dp -> [Text])
-  -> IO (Trace IO dp)
-mkDataPointTracer trDataPoint namesFor = do
-    let tr = NT.contramap DataPoint trDataPoint
-    pure $ withNamesAppended namesFor tr
-```
+As mentioned earlier, trace backends serve as the final destinations for all traces once they have undergone trace interpretation, resulting in metrics and messages. The system defines three trace backends:
 
-Also, [there is a document](https://github.com/input-output-hk/cardano-node-wiki/wiki/cardano-node-and-DataPoints:-demo)
-describing how to accept DataPoints from an external process. [`demo-acceptor`](https://github.com/intersectmbo/cardano-node/blob/master/cardano-tracer/demo/acceptor.hs) application allows to ask for particular DataPoint by its name and display its value.
+1. __Standard Tracer:__ This is the fundamental standard output tracer. Notably, it can accept both regular and confidential traces. It's important to construct only one standard tracer in any application, as attempting to create a new one will result in an exception.
 
-# Integration and implementation in the node
+    ```haskell
+    standardTracer :: forall m. (MonadIO m)
+      => m (Trace m FormattedMessage)
+    ```
 
+2. __Trace-Forward Tracer:__ This is a network-only sink dedicated to forwarding messages using typed protocols over TCP or local sockets. It exclusively handles public traces and should be instantiated only once per application.
```

Comment:
Is forwarding over TCP an option, or has that been removed? A quick look of how I'd configure this came up empty...

Reply:
I can't see the relation of your question to this part. The configuration of the forwarder should be part of the other two documents: "cardano-tracer manual" and "quick start of new tracing for admins"